avocado/core/mux.py

```diff
@@ -132,7 +132,7 @@ def update_defaults(self, defaults):
         combination.merge(self.root)
         self.variants = MuxTree(combination)
 
-    def to_str(self, summary=0, variants=0, use_utf8=False):
+    def to_str(self, summary, variants, extra):
```

Comment:
I actually did but then I worked on avocado-vt and said myself that English version is `extra`. So the question is whether the plugin writers will be more human or python. I have no direct preference here so if you prefer that, I'll change it.

Also about the `**` vs `argument` I thought it'd be clearer to hand over a custom dictionary with values rather than using kwargs. It'd better separate those two, but again, no preference here (the first version was actually like that)